packages/api-clients/open-api/bffApi.yml

```
@@ -3434,7 +3434,7 @@ paths:
       tags:
         - eservices
       summary: Add e-service interface for provided template
```

Same change done in catalogApi
```suggestion
      summary: Add EService template instance interface
```

---

handled with [c212232](https://github.com/pagopa/interop-be-monorepo/pull/1620/commits/c212232e6615a52174a8b71226d1609061ae7316)